GitUI/CommandsDialogs/FormDiff.Designer.cs

```diff
@@ -166,15 +170,39 @@ private void InitializeComponent()
             // 
             // ckCompareToMergeBase
             // 
+            this.ckCompareToMergeBase.Anchor = AnchorStyles.Left;
             this.ckCompareToMergeBase.AutoSize = true;
-            this.ckCompareToMergeBase.Location = new System.Drawing.Point(3, 51);
             this.ckCompareToMergeBase.Name = "ckCompareToMergeBase";
-            this.ckCompareToMergeBase.Size = new System.Drawing.Size(141, 17);
-            this.ckCompareToMergeBase.TabIndex = 8;
-            this.ckCompareToMergeBase.Text = "Compare to merge base";
+            this.ckCompareToMergeBase.TabIndex = 9;
+            this.ckCompareToMergeBase.Text = "Compare to merge &base";
+            this.ckCompareToMergeBase.UseMnemonic = true;
             this.ckCompareToMergeBase.UseVisualStyleBackColor = true;
             this.ckCompareToMergeBase.CheckedChanged += new System.EventHandler(this.ckCompareToMergeBase_CheckedChanged);
             // 
+            // btnCompareDirectoriesWithDiffTool
+            // 
+            this.btnCompareDirectoriesWithDiffTool.Anchor = AnchorStyles.Left;
+            this.btnCompareDirectoriesWithDiffTool.AutoSize = true;
```

Comment:
Need to add `AutoSizeMode = GrowAndShrink`

Reply:
✔️